### DISCUSSION TITLE
Dedicated Shop Page for "Get More Tokens" Packs

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#347] [Suggestions]
So in Ark, the in-game modded shop interface has a slight problem where clicking on the "get more tokens" button will take a user to the community web shop; specifically the "[supporter](https://nyctark.com/supporter-tier)" page. Unfortunately the page is a catch-all for all shop packs that use cash (paypal/stripe/etc) checkouts, not just specifically for packs that sell tokens to players. 

It'd be great if there could be organization similar to the token shop packs available for "supporter" packs, or if there can be a way to designate a page entirely for purchasing tokens, without the distraction of other packs that might be there.

**Comment by mvaessen** [comment]
Thanks for your feedback!
This will be addressed in #357, which will add the option to select an alternative page
